genesis/process/intermediate/standardDelegationProcessor.go

```diff
@@ -171,8 +166,8 @@ func (sdp *standardDelegationProcessor) setDelegationStartParameters(smartContra
 	return nil
 }
 
-func (sdp *standardDelegationProcessor) executeSetNumNodes(numNodes int, sc genesis.InitialSmartContractHandler) error {
-	setNumNodesTxData := fmt.Sprintf("%s@%x", setNumNodesFunction, numNodes)
+func (sdp *standardDelegationProcessor) executeSetServiceFee(serviceFee int, sc genesis.InitialSmartContractHandler) error {
```

Comment:
wasn't this supposed to be set in constructor ? why is this needed separately.

Reply:
removed, remained from initial testing.